### PR TITLE
various man page fixes

### PR DESCRIPTION
**Pull request by poettering**
Fixes: #10338 #9681 #9857 #9870 #10018 #10433 #10529

### CODE REVIEW

File: man/systemctl.xml [hunk 1]
```diff
@@ -1058,6 +1058,12 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
             terminating abnormally or timing out), it will automatically enter the <literal>failed</literal> state and
             its exit code and status is recorded for introspection by the administrator until the service is
             stopped/re-started or reset with this command.</para>
+
+            <para>In addition of resetting the <literal>failed</literal> state of a unit it also resets various other
```
Comment (cdown) [inline man/systemctl.xml]: should be "in addition to"

File: man/systemd.xml [hunk 2]
```diff
@@ -392,6 +392,25 @@
     <citerefentry><refentrytitle>systemd.special</refentrytitle><manvolnum>7</manvolnum></citerefentry>
     for details about these target units.</para>
 
+    <para>systemd only keeps a minimal set of units loaded into memory. Specifically, only units are kept loaded into
```
Comment (cdown) [inline man/systemd.xml]: "Specifically, the only units that are kept loaded into memory are those for which at least one of the following conditions is true"

**Comment by poettering** [comment]
@cdown thanks, fixed both issues and force pushed a new version. ptal!

File: man/systemctl.xml [hunk 3]
```diff
@@ -1058,6 +1058,12 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
             terminating abnormally or timing out), it will automatically enter the <literal>failed</literal> state and
             its exit code and status is recorded for introspection by the administrator until the service is
             stopped/re-started or reset with this command.</para>
+
+            <para>In addition to resetting the <literal>failed</literal> state of a unit it also resets various other
+            per-unit properties: the start rate limit counter of all unit types is reset to zero, as is the restart
+            counter of service units. Thus, if a unit's start limit (as configured with
+            <varname>StartLimitIntervalSec=</varname>/<varname>StartLimitBurst=</varname>) is hit and thus refuses to
+            be started again, use this command to make the unit startable again.</para>
```
Comment (keszybz) [inline man/systemctl.xml]: "thus" → "the unit". (The problem with this sentence is that the subject is "unit's start limit", so without a pronoun, the second part of the sentence would mean "the start limit refuses to be started again").

File: man/systemctl.xml [hunk 4]
```diff
@@ -1085,6 +1091,10 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
             <option>--after</option>, <option>--before</option>
             may be used to change what types of dependencies
             are shown.</para>
+
+            <para>Note that this command only lists units currently loaded into memory by the service manager. In
+            particular, this command is not suitable to get a comprehensive list at all reverse dependencies on a
+            specific unit, as it won't list the dependencies effected by units currently not loaded.</para>
```
Comment (keszybz) [inline man/systemctl.xml]: effected → declared ?

File: man/systemd.socket.xml [hunk 5]
```diff
@@ -94,6 +94,18 @@
     socket passing (i.e. sockets passed in via standard input and
     output, using <varname>StandardInput=socket</varname> in the
     service file).</para>
+
+    <para>All network sockets allocated through <filename>.socket</filename> units are allocated in the host's network
+    namespace (see <citerefentry
+    project='man-pages'><refentrytitle>network_namespaces</refentrytitle><manvolnum>7</manvolnum></citerefentry>). This
+    does not mean however that the service activated as effect of a configured socket has to be part of the host's
```
Comment (keszybz) [inline man/systemd.socket.xml]: "activated as effect of a configured socket" → "activated by a configured socket" ?

File: man/systemd.unit.xml [hunk 6]
```diff
@@ -990,12 +990,13 @@
...
-        respected. A failing condition will not result in the unit being moved into a failure state. The condition is
-        checked at the time the queued start job is to be executed. Use condition expressions in order to silently skip
-        units that do not apply to the local running system, for example because the kernel or runtime environment
-        doesn't require its functionality. Use the various <varname>AssertArchitecture=</varname>,
-        <varname>AssertVirtualization=</varname>, … options for a similar mechanism that puts the unit in a failure
-        state and logs about the failed check (see below).</para>
+        respected. A failing condition will not result in the unit being moved into the <literal>failed</literal>
+        state. The condition is checked at the time the queued start job is to be executed. Use condition expressions
+        in order to silently skip units that do not apply to the local running system, for example because the kernel
+        or runtime environment doesn't require its functionality. Use the various
```
Comment (keszybz) [inline man/systemd.unit.xml]: "its"  → "their" (to match "units").

File: man/systemd.xml [hunk 7]
```diff
@@ -392,6 +392,25 @@
...
+    <para>systemd only keeps a minimal set of units loaded into memory. Specifically, the only units that are kept
+    loaded into memory are those for which at least one of the following conditions is true:</para>
+
+    <orderedlist>
+      <listitem><para>It is in an active, activating, deactivating or failed state (i.e. in any unit state except for <literal>dead</literal>)</para></listitem>
+      <listitem><para>It has a job queued for it</para></listitem>
+      <listitem><para>It is a dependency of some sort of at least one other unit that is loaded into memory</para></listitem>
+      <listitem><para>It has some form of resource still allocated (e.g. a service unit that is inactive but for which
+      a process is still lingering that ignored the request to be terminated)</para></listitem>
+      <listitem><para>It has been pinned into memory programmatically by an IPC client</para></listitem>
```
Comment (keszybz) [inline man/systemd.xml]: "by an IPC client" → "by a D-Bus call" (IPC client is pretty jargony).

File: man/systemd.xml [hunk 8]
```diff
@@ -392,6 +392,25 @@
...
+    </orderedlist>
+
+    <para>systemd will automatically and implicitly load units from disk — if they are not loaded yet — as soon as
+    operations are requested for them. Thus, in many respects, the fact whether a unit is loaded or not is invisible to
+    clients. Use <command>systemctl list-units --all</command> to comprehensively list all units currently loaded. Any
```
Comment (keszybz) [inline man/systemd.xml:409]: not "currently loaded".

File: man/systemd.exec.xml [hunk 9]
```diff
@@ -987,11 +986,19 @@ StateDirectory=aaa/bbb ccc</programlisting>
...
+        propagation in the other direction is not affected, i.e. mounts created on the host generally appear in the
+        unit processes' namespace, and mounts removed on the host also disappear there too. In particular, note that
+        mount propagation from host to unit will result in unmodified mounts to be created in the unit's namespace,
+        i.e. writable mounts appearing on the host will be writable in the unit's namespace too, even when propagated
+        below a path marked with <varname>ReadOnlyPaths=</varname>! Restricting access with these options hence does
+        not extend to submounts of a directory that are created later on. This means the lock-down offered by that
+        setting is not complete, and does not offer full protection. </para>
+
+        <para>Note that the effect of these settings may be undone by privileged processes. In order to set up an
+        effective sandboxed environment for a unit it is thus recommended to combine these settings with either
```
Comment (sourcejedi) [inline man/systemd.exec.xml:1001]: Looking good, thanks for doing this work.  I like that you re-ordered it, so the two <del>restrictions</del> limitations are written next to each other.

File: man/systemd.xml (hunk 8 above)
Comment (poettering) [reply man/systemd.xml:409]: hmm?

Comment (poettering) [reply man/systemd.xml:409]: the call only shows the loaded units, and that comprehensively, so I am not sure I grok your comment?

**Comment by poettering** [comment]
I force pushed a new version with all points addressed, except for the one which I didn't grok, see above

**Comment by cdown** [comment]
Looks fine to me now, I'll leave it @keszybz to comment on previously mentioned technical aspects though :)

File: man/systemd.xml (hunk 8 above)
Comment (keszybz) [reply man/systemd.xml:409]: My bad.

**Pull request merged**